platform/cli/cli.ts

```diff
@@ -52,4 +53,24 @@ const run = (argv: { [key: string]: any }) => {
   }
 };
 
+export const runWithLegacyKbnServer = (kbnServer: any) => {
+  const root = new Root(
+    getLegacyConfig$(kbnServer),
```

Comment:
@kjbekkelund I've added some initial support for transforming legacy config to a new format, could you please take a look?

It also required some changes to `Duration` and `ByteSize` since in the old Kibana we use numbers for these type of values (either number of ms or bytes). I prefer the new platform approach though (strings like `1mb` etc.) and would get rid of number support once we finish migration. Alternatively `LegacyPlatformConfig` can transform numbers to format new platform understands like `newconfig.timeout: "${oldconfig.timeout}"ms`, but I found it a bit clumsy :)

Reply:
Awesome, I'll take a look now.